### ISSUE TITLE
Docker for Windows installation

### ISSUE THREAD

**Issue by Croydon-Brixton** [csi-greifflab/Absolut#3]
Hi,
Great project - I really liked the paper. I am trying to use Absolut to discretise my own antigens and my current understanding is that this does not work with the Linux version (is that correct?). 

I have thus tried to wrap my head around the Windows installation, but find the given guidelines in `HowToAbsolut` hard to follow. Would it be possible to create a Windows docker image that has a working installation that I could use? 

Alternatively I am happy to help update the installation instructions if you could help guide me through the installation process.

**Comment by petouzeman** [comment]
Dear Simon,
I am mega sorry, I didn't get notification of the issues and I am using another gitlab for step-by-step development, so I oversaw your issue. And thanks for your comment! 
It is actually possible to do everything in Linux (indeed easier). I will update the how to in the following days, and feel free to contact me by mail probertmontp at gmail.com
Have a nice day!

**Comment by petouzeman** [comment]
In case you are still interested in going through the installation, tell me!
